File: kanye/src/part1/exercise11.rs
// function to print average of numbers in an array of numbers
pub fn average(arr: [i32; 10]) {
    let mut sum = 0;
    for i in arr.iter() {
        sum += i;
    }
    let mut average : f32 = sum as f32 / (arr.len() as f32);
    println!("The average of the numbers in the array is {}", average);
}


// function to print average of numbers in an array of numbers variant 1
pub fn average1(arr: [i32; 10]) {
    let mut sum = 0;
    let mut average;
    let mut i = 0;
    while i < arr.len() {
        sum += arr[i];
        i += 1;
    }
    average = sum as f32 / arr.len() as f32;
    println!("The average of the numbers in the array is {}", average);
}

File: Randy-JRA/Exercise_11.cpp
//Challenge 11 is to Calculate the average of the numbers in an array of numbers 
//Example array[1, 3, 9, 15, 90] Expected output:23.6

#include <iostream>
using namespace std;
int main() {
  // we first initialise out array
  int arr[] = {1,3,9,15,90};
  float sizeofarray = 5;
  // initialise sum to zero
  float sum = 0;
  float avrg;
  // for loop runs from 0 to sizeofarray - 1
  for(int i = 0; i < sizeofarray ; i++)
  {
    sum = sum + arr[i];
  }
  avrg = sum/sizeofarray;
  cout << "The average of the elements in the array: " << avrg;
}
